### PR TITLE
shipit_frontend: Display diff between prev & current in code coverage…

### PR DESCRIPTION
**Pull request by La0**
…, refs #318

### CODE REVIEW

File: src/shipit_frontend/src/App/CodeCoverage.elm [hunk 1]
```diff
@@ -164,28 +164,60 @@ viewDirectories directories =
...
             [ td [] [ span [ class "btn btn-link", onClick (SetDirectory (Just path)) ] [ text path ] ]
-            , td [] [ text (toString (toFloat (round (directory.previous * 100)) / 100)) ]
+            , td [] [ text previous ]
             , td [] [ text (toString directory.current) ]
+            , td []
+                [ text
+                    ((if diff > 0 then
+                        "+"
+                      else
+                        ""
```
Comment (marco-c) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Should this be "-"?

**Comment by marco-c** [comment]
Can we make rows with diff < 0 red and rows with diff >= 0 green?

File: src/shipit_frontend/src/App/CodeCoverage.elm [hunk 2]
```diff
@@ -164,28 +164,60 @@ viewDirectories directories =
...
+    toFloat (round (x * 100)) / 100
+
+
 viewDirectory : ( String, Directory ) -> Html Msg
 viewDirectory ( path, directory ) =
     let
+        diff =
+            case directory.previous of
+                Just previous ->
+                    roundFloat (directory.current - previous)
```
Comment (marco-c) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Should this be directory.previous? `previous` is a string, no?

Comment (La0) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: No, previous is a `Float` as directory.previous is a `Maybe Float`

File: src/shipit_frontend/src/App/CodeCoverage.elm (hunk 1 above)
Comment (La0) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: No, the "-" is already displayed for negative values

Comment (marco-c) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Right 😄

File: src/shipit_frontend/src/App/CodeCoverage.elm (hunk 2 above)
Comment (marco-c) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Ah, so `previous` here is not the one defined on line 199, but the destructuring of directory.previous from line 184. Correct?

Comment (La0) [inline src/shipit_frontend/src/App/CodeCoverage.elm]: Yeap :+1:

**Comment by La0** [comment]
The table-danger/table-success style already displays in red/green according to diff values

**Comment by marco-c** [comment]
> The table-danger/table-success style already displays in red/green according to diff values

Good, I had forgotten about that.

**Pull request merged**